### ISSUE TITLE
12am and 4am survival

### ISSUE THREAD

**Issue by sleeping-player** [MathewSachin/Fate-Grand-Automata#310]
I write this as I cannot find this function on issue.

FGO has two different daily counter reset time. At this time, the client is reset in different ways.

At 12am, friend gacha is refilled, and daily dungeon is reset. Connected dungeon at that time slightly blinks after clear.

4am is well-known for daily bonus charge time, and this time is important to the FGO server. unlike 12am, client is reset significantly at 4am.

If i clear the dungeon after 4am, the client exits to the main screen. To access the previous dungeon, I must manually enter the dungeon image into the FGA and let it detect it at 4am.

As this is is the KR situation, international time and behaviour may differ from this.

I'm sorry as I'm sleeping at this time, so actually I don't know what just happened at 4am. I think I'll ask someone to report a video needed.

**Comment by MathewSachin** [comment]
Don't bother for a video. This can't be supported in the app. There's just too many random things that can happen on these rest times. Heck, even a maintenance can happen.
I think it's better to press the buttons on the rest times manually. It's like only once a day 🤷‍♂️

**Comment by sleeping-player** [comment]
Actually this is possible as Korean masters used another automating app on Windows. This supports that feature. (except sudden maintenance of course)

**Comment by MathewSachin** [comment]
Yeah, it's not impossible. But, I have to support 5 different servers. Not really the best use of my time. Maybe you can fork the code or get someone who plays KR to do this?

**Comment by sleeping-player** [comment]
OK. I understand.
And is actually another issue, I sent something. As Paypal could name as actual name, please note as just sleeping-player.

**Comment by MathewSachin** [comment]
Damn, that was a big donation. Thanks!
This really changes things for me. I don't have to worry about apples running out in testing before Gilfest now!
